app/videobridge/server/methods/jitsiSetTimeout.js

```diff
@@ -50,9 +50,7 @@ Meteor.methods({
 
 			return jitsiTimeout || nextTimeOut;
 		} catch (error) {
-			SystemLogger.error('Error starting video call:', error);
-
-			throw new Meteor.Error('error-starting-video-call', error.message);
+			SystemLogger.error('Error starting video call:', error.message);
```

Comment:
But in this case, this will silently fail isn't it? 🤔 so the user won't receive feedback of why it failed? (or SystemLogger takes care of that?)

Reply:
Well, no warnings were being shown to the user (in the UI) even before the changes here -- so removing the `Meteor.Error` made no difference to the user. I guess this `Meteor.Error` should be doing that, shouldn't it?

Reply:
Yeah, I can confirm. User in that case don't get aby feedback now, But admin gets flooded logs with Meteor stack trace.